addons/website_mass_mailing/static/src/js/website_mass_mailing.editor.js

```diff
@@ -155,6 +155,11 @@ options.registry.newsletter_popup = options.registry.mailing_list_subscribe.exte
      * @override
      */
     start: function () {
+        this.$target.on('shown.bs.modal.newsletter_popup_option', () => {
+            this.trigger_up('activate_snippet', {
+                $snippet: this.$target,
+            });
+        });
```

Comment:
I don't understand this, that would need comments?  Is not it already activated once the modal is opened?